src/webgl/p5.Framebuffer.js

```diff
@@ -931,6 +950,238 @@ class Framebuffer {
     callback();
     this.end();
   }
+
+  loadPixels() {
+    const gl = this.gl;
+    const prevFramebuffer = gl.getParameter(gl.FRAMEBUFFER_BINDING);
+    gl.bindFramebuffer(gl.FRAMEBUFFER, this.framebuffer);
+    const colorFormat = this._glColorFormat();
+    this.pixels = readPixelsWebGL(
+      this.pixels,
+      gl,
+      this.framebuffer,
+      0,
+      0,
+      this.width * this.density,
+      this.height * this.density,
+      colorFormat.format,
+      colorFormat.type
+    );
+    gl.bindFramebuffer(gl.FRAMEBUFFER, prevFramebuffer);
+  }
+
+  /**
+   * Get a region of pixels, or a single pixel, from the canvas.
+   *
+   * Returns an array of [R,G,B,A] values for any pixel or grabs a section of
+   * an image. If the Framebuffer has been set up to not store alpha values, then
+   * only [R,G,B] will be returned. If no parameters are specified, the entire
+   * image is returned.
+   * Use the x and y parameters to get the value of one pixel. Get a section of
+   * the display window by specifying additional w and h parameters. When
+   * getting an image, the x and y parameters define the coordinates for the
+   * upper-left corner of the image, regardless of the current <a href="#/p5/imageMode">imageMode()</a>.
+   *
+   * @method get
+   * @param  {Number}         x x-coordinate of the pixel
+   * @param  {Number}         y y-coordinate of the pixel
+   * @param  {Number}         w width
+   * @param  {Number}         h height
+   * @return {p5.Image}       the rectangle <a href="#/p5.Image">p5.Image</a>
```

Comment:
Does this return a p5.Image in all cases though? Doesn't providing just x & y give a single pixel value? 

For the return description instead of "the rectangle" can we say something like: "The rectangular area of pixels defined by the input parameters.".

Reply:
> Does this return a p5.Image in all cases though? Doesn't providing just x & y give a single pixel value?

Right, I think this is a limitation of our docs system. The overload with just x and y provided has `@return {Number[]}`, but it doesn't render a return type for each option in the docs (maybe it just picks the first one?) Maybe for now the best compromise is to keep all the data here for each overload and mention in the description of the method what the output will be for each case?

Reply:
sgtm!